compliance/cis_k8s/rules/cis_1_1_13/data.yaml

```diff
@@ -0,0 +1,45 @@
+metadata:
+  id: 1aa16467-18ce-5ba6-8e6a-1f6b69acbfa0
+  name: Ensure that the admin.conf file permissions are set to 644 or more
+    restrictive (Automated)
+  profile_applicability: |
+    * Level 1 - Master Node
+  description: >
+    Ensure that the `admin.conf` file has permissions of `644` or more
+    restrictive.
+  rationale: >
+    The `admin.conf` is the administrator kubeconfig file defining various
+    settings for the administration of the cluster. You should restrict its file
+    permissions to maintain the integrity of the file. The file should be
+    writable by only the administrators on the system.
+  audit: |
+    Run the following command (based on the file location on your system) on the
```

Comment:
There are new line that affects how the text would be presented on the UI.
Might be needed a fix here.

Reply:
Pushed an align commit.
I've implemented all of the rules similar to how they are being visualized in the benchmark PDF.